lib/services/project-properties-service.ts

```diff
@@ -40,16 +40,6 @@ export class ProjectPropertiesService implements IProjectPropertiesService {
 
 		updated = frameworkProject.completeProjectProperties(properties);
```

Comment:
not part of the current changes, but I've noticed that before this line you have one if, in which you are setting updated = true; In case completeProjectProperties returns false, you'll overwrite this updated value